### PR TITLE
Order placement #31

### PR DESCRIPTION
**Pull request by TyMazey**
users can now place orders/create orders.
updated other test that were not using number to currency in view.

### CODE REVIEW

File: app/controllers/users/orders_controller.rb [hunk 1]
```diff
@@ -7,4 +7,17 @@ def index
...
+  def create
+    order = Order.new(user: current_user)
+    if order.save
+      @cart.contents.each do |item_id, quantity|
+        current_price = Item.find(item_id).price * quantity
+        order.order_items.create(item_id: item_id, quantity: quantity, current_price: current_price)
+      end
+      session[:cart].clear
+      flash[:success] = 'Your order has been placed.'
+      redirect_to profile_path
```
Comment (n-flint) [inline app/controllers/users/orders_controller.rb]: possible issue - profile_path should be profile_orders_path
discrepancy in user story between /profile, and wanting to view the orders /profile/orders

File: db/migrate/20190212183022_change_order_items.rb [hunk 2]
```diff
@@ -1,6 +0,0 @@
-class ChangeOrderItems < ActiveRecord::Migration[5.1]
-  def change
-    add_column :order_items, :current_price, :float
-    add_column :order_items, :quantity, :integer
-  end
-end
```
Comment (n-flint) [inline db/migrate/20190212183022_change_order_items.rb:6]: Did you mean to delete this migration?

File: spec/features/user_cart_spec.rb [hunk 3]
```diff
@@ -225,12 +225,16 @@
...
+      expect(current_path).to eq(profile_path)
       expect(page).to have_content('Cart(0)')
       expect(page).to have_content('Your order has been placed.')
-      within '.orders' do
-        expect(page).to have_content('Order: ')
-      end
+
+      click_link 'Orders'
+
+      expect(page).to have_content('ID: ')
```
Comment (n-flint) [inline spec/features/user_cart_spec.rb:231]: I think on line 234, and 235 we are missing the actual values for Id: and Order Placed:

**Comment by Vjp888** [comment]
Changes were made and got clarity for pathing issues. Both Noah and I approve this for merge.

**Comment by renecasco** [comment]
If changes were made it seems like we're good to merge!

**Pull request merged**